edx/edx-enterprise
# ENT-767 Check embargo restrictions when rendering enterprise landing pages.

Redirect the user to the embargo restriction page if their access
to the given course run/program is restricted.

**JIRA:** https://openedx.atlassian.net/browse/ENT-767

**Testing Instructions:**
1. Visit https://business.sandbox.edx.org/enterprise/2b5a2956-7746-4fc9-9587-bc887ba45973/course/course-v1:edX+DemoX+Demo_Course/enroll/?catalog=cff09318-1358-4703-a277-bc2596480ed5&utm_medium=enterprise&utm_source=pied-piper.
2. Register a new account. Choose "Canada" as your country.
3. Verify that the Embargo restriction message page is displayed.

![screen shot 2017-12-05 at 12 43 27 pm](https://user-images.githubusercontent.com/224634/33621789-f4f0774e-d9b9-11e7-9644-779bcf11e526.png)
